project/plugins.sbt

```diff
@@ -1,11 +1,13 @@
-addSbtPlugin("com.typesafe.play"        % "sbt-plugin"          % "2.8.16")
+addSbtPlugin("com.typesafe.play"        % "sbt-plugin"          % "2.8.18")
 addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8-scaffold" % "0.13.1")
 addSbtPlugin("com.eed3si9n"             % "sbt-buildinfo"       % "0.11.0")
 addSbtPlugin("org.scalameta"            % "sbt-scalafmt"        % "2.4.4")
 addSbtPlugin("de.heikoseeberger"        % "sbt-header"          % "5.6.0")
 addSbtPlugin("io.github.irundaia"       % "sbt-sassify"         % "1.5.2")
 addSbtPlugin("com.github.sbt"           % "sbt-release"         % "1.1.0")
 addSbtPlugin("org.scoverage"            % "sbt-scoverage"       % "2.0.6")
+addSbtPlugin("com.codecommit"           % "sbt-github-packages" % "0.5.3")
+addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.16")
```

Comment:
Why do we need this?

Reply:
There was an issue with conflicting versions of the SLF4J logging provider; this SBT dependency helped track down the issue. It is unused now and I have deleted this line plugin.

Reply:
OK.